templates/payments/payout_group.html

```diff
@@ -21,11 +21,12 @@
                 </p>
                 <p>
                     <form action="/payments/payout_group_processed" method="post" id="submit_form_all_completed">
+                        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
                         <input type="hidden" name="token" value="{{group.token}}" />
                         <input type="button" value="Set all completed" data-action="set all completed" data-form="submit_form_all_completed" data-toggle="modal" data-target="#confirm_submit" class="btn btn-primary submit_button" />
                     </form>
                 </p>
             </div>
         </div>
     </div>
-{% endblock %}
```

Comment:
remove irrelevant change

Reply:
> remove irrelevant change

not sure what this means. i can see the endblock

Reply:
re-added the "no newline at the end of file".